tests/integration/test_specfile.py

```diff
@@ -0,0 +1,111 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import datetime
+import shutil
+import subprocess
+from pathlib import Path
+
+import pytest
+from flexmock import flexmock
```

Comment:
As far as I can see, `flexmock` is not in `setup.cfg`, you should probably add it there

Reply:
Will do.